tests/warper.py

```diff
@@ -35,18 +35,19 @@
 import lsst.utils.tests as utilsTests
 import lsst.pex.logging as pexLog
 import lsst.pex.policy as pexPolicy
+import lsst.pex.exceptions as pexExcept
 
 VERBOSITY = 0                       # increase to see trace
 
 pexLog.Debug("lsst.afw.math", VERBOSITY)
 
 try:
-    afwDataDir = lsst.utils.getPackageDir("afwdata")
-except Exception:
-    warnings.warn("skipping all tests because afwdata is not setup")
+    afwdataDir = lsst.utils.getPackageDir("afwdata")
+except pexExcept.NotFoundError:
+    afwdataDir = None
     dataDir = None
```

Comment:
It get's used later in getSwarpedImage(). I wanted to keep as much as in to avoid introducing new bugs.